src/dhcpmon/src/subdir.mk

```diff
@@ -0,0 +1,27 @@
+# Add inputs and outputs from these tool invocations to the build variables 
+C_SRCS += \
+../src/dhcp_device.c \
+../src/dhcp_devman.c \
+../src/dhcp_mon.c \
+../src/main.c 
+
+OBJS += \
+./src/dhcp_device.o \
+./src/dhcp_devman.o \
+./src/dhcp_mon.o \
+./src/main.o 
+
+C_DEPS += \
+./src/dhcp_device.d \
+./src/dhcp_devman.d \
+./src/dhcp_mon.d \
+./src/main.d 
+
+
+# Each subdirectory must supply rules for building sources it contributes
+src/%.o: ../src/%.c
+	@echo 'Building file: $<'
+	@echo 'Invoking: GCC C Compiler'
+	gcc -O3 -g3 -Wall -c -fmessage-length=0 -MMD -MP -MF"$(@:%.o=%.d)" -MT"$(@)" -o "$@" "$<"
```

Comment:
Probably change gcc to $(CC)

Reply:
ACK

Reply:
Looks like this was not done as part of your recent commit addressing comments. Is there an issue, or just missed?